README.md

```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
 }
 ```
 
-In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers, initialState)` function to provide them to Angular's injector:
+In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers)`
```

Comment:
Complains about not knowing the module for the `FocusTrap` component which seems to be an unregistered component published in @angular/material.